draft-bbs-signatures.md

```diff
@@ -421,25 +421,23 @@ Procedure:
 
 3. generators =  (H_s || H_d || H_1 || ... || H_L)
 
-4. domain = OS2IP(hash(PK || L || generators || Ciphersuite_ID || header)) mod q
+4. domain = hash_to_scalar((PK || L || generators || Ciphersuite_ID || header), 1)
 
-5. if domain is 0, abort
+5. h = xof(SK  || domain || msg_1 || ... || msg_L)
 
-6. h = xof(SK  || domain || msg_1 || ... || msg_L)
+6. for element in (e, s) do
 
-7. for element in (e, s) do
+7.      element = OS2IP(h.read(xof_no_of_bytes)) mod r
```

Comment:
I think we should update computing e and s too as #135 is un-likely to be merged in its current form

Reply:
Yeap! Done!